circle.yml

```diff
@@ -1,9 +0,0 @@
-machine:
-  node:
-    version: 4.4.3
```

Comment:
Without this Circle is running it's tests on node 0.10.33. Whereas next-article is running on 4.3.1. Maybe advisable to leave this part of the circle.yml in there.